#!/usr/bin/python3
def raise_exception_msg(message=""):
    """raisees a name exception"""
        raise NameError(message)
